source/_integrations/bouncie.markdown

```diff
@@ -0,0 +1,44 @@
+---
+title: Bouncie
+description: Instructions on how to integrate Bouncie within Home Assistant.
+ha_category:
+  - Car
+ha_release: 2021.2
+ha_iot_class: Cloud Polling
+ha_config_flow: true
+ha_codeowners:
+  - '@mandarons'
+ha_domain: bouncie
+ha_platforms:
+  - sensor
+ha_integration_type: integration
+---
+
+The `bouncie` integration uses the [Bouncie](https://www.bouncie.dev/) web service
+as a source for monitoring your car(s).
+
+## Setup
+
+To generate an Bouncie application credentials -
+
+1. Go to the [Bouncie Developer Page](https://bouncie.dev/login/) page
+2. Login with your `bouncie.com` credentials
+3. Click on `+ ADD APPLICATION` 
+4. Provide a `name` for application e.g. Home Assistant Integration
+5. Provide a unique `client_id` e.g. alphanumeric random string
+6. Provide redirect URL to be your Home Assistant URL
```

Comment:
The redirect URL should use the My Home Assistant service we provide. See, for example, the Spotify integration as an example.